.eslintrc.js

```diff
@@ -37,4 +37,26 @@ module.exports = {
     'testing-library/prefer-screen-queries': 0,
     'turbo/no-undeclared-env-vars': 0,
   },
+
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      parser: '@typescript-eslint/parser',
+      plugins: ['@typescript-eslint'],
+      extends: [
+        'eslint:recommended',
+        'prettier',
+        'plugin:prettier/recommended',
+        'plugin:import/errors',
+        'plugin:import/warnings',
+        'plugin:import/typescript',
+        'plugin:cypress/recommended',
+        'plugin:ui-testing/cypress',
+        'plugin:testing-library/react',
+        'turbo',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+      ],
+    },
+  ],
```

Comment:
We don't have TS files. They're TS files in the tools folder but we've skipped lining in that folder. Why are we adding this configuration?

Reply:
`index.d.ts` which defines types of all our commands.

Reply:
We don't need to configure ESLint for specifically one TS file. Plus we don't have any plans to transition to TS. I feel it is not required.

Reply:
the files exists in the repository and we will be adding code to it in the future. i think its best that we add support for typescript eslint now. rather than lets seperately

Reply:
Yeah, we will be updating that file. But adding this whole config just for one TS file isn't a overhead?